spec/spec_helper.rb

```diff
@@ -3,23 +3,50 @@
   SimpleCov.start
 end
 
-Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
-Dir[File.join(__dir__, "support/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/shared/**/*.rb")].sort.each { |f| require f }
+Dir[File.join(__dir__, "support/**/*.rb")].sort.each { |f| require f }
```

Comment:
This is interesting...we should probably add this to the core generator for consistency - @agrare ?

Reply:
Do we need require order to be guaranteed?  Doesn't seem to be needed for core or any providers currently.

Reply:
Probably not, but it's more deterministic just in case.